src/Features/VisualBasic/Portable/CodeFixes/RemoveUnnecessaryCast/RemoveUnnecessaryCastCodeFixProvider.RemoveUnnecessaryCastFixAllProvider.vb

```diff
@@ -1,38 +0,0 @@
-﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-Imports System.Threading
-Imports Microsoft.CodeAnalysis.CodeFixes
-Imports Microsoft.CodeAnalysis.Options
-Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
-
-Namespace Microsoft.CodeAnalysis.VisualBasic.CodeFixes.RemoveUnnecessaryCast
-    Partial Friend Class RemoveUnnecessaryCastCodeFixProvider
-        Inherits CodeFixProvider
-
-        Private Class RemoveUnnecessaryCastFixAllProvider
-            Inherits BatchSimplificationFixAllProvider
```

Comment:
no need for this.  SyntaxEditorBasedCodeFixProvider already has its own provided fix-all provider.